src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigatorTest.java

```diff
@@ -459,14 +505,14 @@ public void register_hooks() throws Exception {
 
     @Test
     public void given__instance__when__setTraits_empty__then__traitsEmpty() {
-        BitbucketSCMNavigator instance = new BitbucketSCMNavigator("test");
+        BitbucketSCMNavigator instance = new BitbucketSCMNavigator("test", "PK");
```

Comment:
And the original Constructor works too?  Just thinking about other plugins.

Reply:
I've decided to go with setProjectKey and leave constructor as they were.